integration/tests/tracer/test_tracer_debug_trace_transactions_callTracer.py

```diff
@@ -0,0 +1,501 @@
+    
+    def test_callTracer_call_contract_from_other_contract_revert_with_assert(self, caller_contract):
+        sender_account = self.accounts[0]
+        contractTwo = caller_contract[0]
+        address_of_contractOne = caller_contract[1]
+
+        tx = self.web3_client.make_raw_tx(from_=sender_account)
+        instruction_tx = contractTwo.functions.callContactRevertWithAssertFalse(address_of_contractOne).build_transaction(tx)
+        receipt = self.web3_client.send_transaction(sender_account, instruction_tx)
+
+        tracer_params = { "tracer": "callTracer", "tracerConfig": { "withLog": True } }
+
+        wait_condition(
+            lambda: self.tracer_api.send_rpc(
+                method="debug_traceTransaction", params=[receipt["transactionHash"].hex(), tracer_params]
+            )["result"]
+            is not None,
+            timeout_sec=120,
+        )
+
+        response = self.tracer_api.send_rpc(
+            method="debug_traceTransaction", params=[receipt["transactionHash"].hex(), tracer_params]
+        )
+
+        address_to = instruction_tx["to"].lower()
+        reason = f"Insufficient balance for transfer, account = {address_to}, chain = 111, required = 1"
+        expected_response = self.fill_expected_response(instruction_tx, receipt, logs=True, revert=True, revert_reason=reason)
+        self.assert_response_contains_expected(expected_response, response)
+
+    def test_callTracer_call_contract_from_other_contract_revert(self, caller_contract):
+        sender_account = self.accounts[0]
+        contractTwo = caller_contract[0]
+        address_of_contractOne = caller_contract[1]
+
+        tx = self.web3_client.make_raw_tx(from_=sender_account)
+        instruction_tx = contractTwo.functions.callContactRevert(address_of_contractOne).build_transaction(tx)
+        receipt = self.web3_client.send_transaction(sender_account, instruction_tx)
+
+        tracer_params = { "tracer": "callTracer", "tracerConfig": { "withLog": True } }
+
+        wait_condition(
+            lambda: self.tracer_api.send_rpc(
+                method="debug_traceTransaction", params=[receipt["transactionHash"].hex(), tracer_params]
+            )["result"]
+            is not None,
+            timeout_sec=120,
+        )
+
+        response = self.tracer_api.send_rpc(
+            method="debug_traceTransaction", params=[receipt["transactionHash"].hex(), tracer_params]
+        )
+
+        address_to = instruction_tx["to"].lower()
+        reason = f"Insufficient balance for transfer, account = {address_to}, chain = 111, required = 1"
+        expected_response = self.fill_expected_response(instruction_tx, receipt, logs=True, revert=True, revert_reason=reason)
+        self.assert_response_contains_expected(expected_response, response)
+    
+    def test_callTracer_call_contract_from_other_contract_revert_with_require(self, caller_contract):
+        sender_account = self.accounts[0]
+        contractTwo = caller_contract[0]
```

Comment:
let's use snake_case for names as everywhery

Reply:
fixed